haas/ext/switches/_console.py

```diff
@@ -73,6 +73,9 @@ def disable_port(self):
     def disconnect(self):
         """End the session. Must be at the main prompt."""
 
+    def _set_native(self, old_native, network_id, interface):
```

Comment:
You shoud make it more clear that this method is intended to be overridden -- I spend a bit of time puzzling as to why there was an `interface` parameter that wasn't used. Most of the other methods that are overridden also don't actually have implementations (empty bodys).